services/movie.py

```diff
@@ -0,0 +1,37 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__in=actors_ids)
```

Comment:
Do not override your viriable , you could just filter your query in some condition : )